src/scss/components/switch/_index.scss

```diff
@@ -109,13 +98,6 @@ hx-switch-control > input[type="checkbox"]:checked ~ label {
   }
 }
 
-.switch {
-  position: relative;
-  display: inline-block;
-  width: 4.25rem;
-  height: 2.125rem;
-}
-
```

Comment:
Moved this styling into `hx-switch-control`.  This should allow implementors to use:

```js
<hx-switch-control> // removed class="switch"
  <input type="checkbox" id="switchYesNoTest" />
  <label for="switchYesNoTest">
    <hx-switch
      onlabel="yes"
      offlabel="no"
      aria-labelledby="switchYesNoTest">
    </hx-switch>
  </label>
</hx-switch-control>
```